17_kata.py

```diff
@@ -0,0 +1,15 @@
+# https://www.codewars.com/kata/54bb6f887e5a80180900046b/train/python
+
+def longest_palindrome(s):
+    if len(s) > 1:
```

Comment:
You use the `len(s)` multiple (4) times within this block of code. To make it look nicer, but also to make code more efficient, you can define a variable, that would be equal to `len(s)` and then use the variable instead of module `len` many times. 

For example: `string_length = len(s)`